src/rabbit_node_monitor.erl

```diff
@@ -414,7 +414,11 @@ handle_cast({check_partial_partition, Node, Rep, NodeGUID, MyGUID, RepGUID},
                    fun () ->
                            case rpc:call(Node, rabbit, is_running, []) of
                                {badrpc, _} -> ok;
-                               _           -> cast(Rep, {partial_partition,
+                               _           ->  
+				   rabbit_log:warning("Unexpected running node:"
```

Comment:
Yes, as first choice we wrote a message similar to  yours:

> Disconnection from node ~p but the node is still running

I decided to change it, because this call checks if the node `x` is running.  It does not receive the `DOWN` message. 

BTW maybe for the user is most clear another kind of the message. I'm going to change it